src/webhook_launcher/processes/VCSCOMMIT_BUILD

```diff
@@ -30,6 +30,12 @@ Ruote.process_definition 'vcscommit' do
     # This should be conditional though
     create_project
     trigger_service
+    # We wait for the source before checking the build results as
+    # if the service is still running the is_repo_published can
```

Comment:
The prose here feels off "resulst as if"? Maybe:  "We wait for the source before checking build results as is_repo_published can fail if the service is still running."